<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Win_Login</class>
 <widget class="QDialog" name="Win_Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="BTN_Cancel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_Accept">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Entrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="STC_ImagenLogin">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>141</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="STC_Login">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Usuario</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EDT_Login">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="EDT_Pass">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="STC_Pass">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Contrase√±a</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
